meta-intel-extras/recipes-support/grub/grub-efi_%.bbappend

```diff
@@ -13,9 +13,9 @@ do_install_append() {
     install -d ${D}${datadir}
     install -d ${D}${systemd_system_unitdir}
 
-    install -m 0755 ${WORKDIR}/grubenv ${D}${datadir}
+    install -m 0644 ${WORKDIR}/grubenv ${D}${datadir}
     install -m 0755 ${WORKDIR}/grubenv-copy.sh ${D}${bindir}
```

Comment:
grubenv-copy.sh also needs the same permissions

Reply:
It is a executable, so no, it has to have x permission set.